OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -624,14 +624,16 @@ void trackOpenInBrowserBannerEvent(@NonNull String eventName, @NonNull String bi
      * @param componentId Component id of course unit
      * @param elapsedTime time in milliseconds that app took to perform the action(price load, refresh course content)
      * @param error       error that app sending in get help email
-     * @param actionTaken action taken on the error dialog
+     * @param errorAction action taken on the error dialog
+     * @param alertAction action taken on the alert dialog
      * @param screenName  Screen name on which event is triggered
      */
     void trackInAppPurchasesEvent(@NonNull String eventName, @NonNull String biValue,
-                                  @NonNull String courseId, boolean isSelfPaced,
-                                  @Nullable String price, @Nullable String componentId,
-                                  long elapsedTime, @Nullable String error,
-                                  @Nullable String actionTaken, @NonNull String screenName);
+                                  @Nullable String courseId, boolean isSelfPaced,
+                                  @Nullable String flowType, @Nullable String price,
+                                  @Nullable String componentId, long elapsedTime,
+                                  @Nullable String error, @Nullable String errorAction,
+                                  @Nullable String alertAction, @Nullable String screenName);
```

Comment:
we should merge `alertAction`, `errorAction` to minimize the confusion. and add conditions in the implementation use update the key in analytics data accordingly.